src/ccsr/src/client.rs

```diff
@@ -86,11 +85,11 @@ impl Client {
         })
     }
 
-    /// Gets the latest schema for the specified subject as well as all other subjects referenced
-    /// by that subject (recursively).
+    /// Gets the latest version of the specified subject as well as all other
+    /// subjects referenced by that subject (recursively).
     ///
     /// The dependencies are returned in alphabetical order by subject name.
-    pub async fn get_all_subjects(
+    pub async fn get_subject_with_references(
```

Comment:
since this is self-acknowledged as a bikeshedding PR (lol) maybe `get_subject_and_references`? `with` makes me think it's subjects with the references field filled out or only returns subjects that have references.

Reply:
Ha! I actually had the same thought I submitted the PR but didn't bother going back through CI. Done as you suggest.